reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -150,11 +170,12 @@ private Statistics processRow(int it, Set<String> columns, List<Map<String, Stri
       return summary;
    }
 
-   private void addRepresentations(Statistics summary, Map<String, String> rowData, Statistics statistics, String operationName) {
-      //Map.Entry<Integer, List<Statistics>> slaveStats = ; Map.Entry<String, OperationStats> os = ;
+   private void addRepresentations(Statistics summary, Map<String, String> rowData, Statistics statistics,
+         String operationName) {
       DefaultOutcome defaultOutcome = statistics.getRepresentation(operationName, DefaultOutcome.class);
       if (defaultOutcome != null) {
-         if (defaultOutcome.requests == 0) return;
+         if (defaultOutcome.requests == 0)
+            return;
```

Comment:
I am saying I don't, but I will not be starting a flamewar over that (that is where these discussions generally end)